openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -96,6 +96,8 @@ class AccountView(APIView):
               requiring parental consent.
             * username: The username associated with the account.
             * year_of_birth: The year the user was born, as an integer, or null.
+            * account_privacy: The user's setting for sharing her personal
+              profile. Possible values are "all_users" or "private".
```

Comment:
@nasthagiri With current preference behaviour; 
-if an empty string is passed as `account_privacy`; field error is raised
-if null/None is passed; it deletes existing `account_privacy` entry